examples/asr/asr_chunked_inference/ctc/speech_to_text_buffered_infer_ctc.py

```diff
@@ -161,6 +183,32 @@ def autocast():
         )
         return cfg
 
+    # Setup decoding strategy
+    if hasattr(asr_model, 'change_decoding_strategy'):
+        if not isinstance(asr_model, EncDecCTCModel) and not isinstance(asr_model, EncDecHybridRNNTCTCModel):
+            raise ValueError(
+                "The script supports ctc model and hybrid model with ctc decodng! use rnnt/speech_to_text_buffered_infer_rnnt.py for other conditions."
+            )
+
+        else:
+            if cfg.compute_langs:
```

Comment:
If compute_langs is not supported, why do we have it in the config?

Reply:
1. I left it because `# TODO: Support compute_langs in CTC eventually`
2. The user could possibly pass `compute_langs` due to the habits in `transcribe_speech`  or `speech_to_text_buffered_infer_rnnt` and if we don't have it in config, it could throw an error / user think they use compute_langs=true but it's not the case since it's not supported